lib/benchee/formatters/console.ex

```diff
@@ -76,6 +78,16 @@ defmodule Benchee.Formatters.Console do
       end)
   end
 
+  @doc """
+  Takes the output of `format/1` and writes that to the console.
+  """
+  @spec write(any) :: :ok | {:error, String.t}
+  def write(output) do
+    IO.write(output)
+  rescue
+    _ -> {:error, "Unknown Error"}
```

Comment:
The documentation for `IO.write` wasn't very good. It just says it "returns :ok if it succeeds", but gives no information on what happens if it were to fail, so I had to be really broad here.